plugin/src/main/scala/scoverage/ScoveragePlugin.scala

```diff
@@ -230,8 +231,8 @@ class ScoverageInstrumentationComponent(
     ): Tree = {
       safeSource(tree) match {
         case None =>
-          reporter.echo(
-            s"[warn] Could not instrument [${tree.getClass.getSimpleName}/${tree.symbol}]. No pos."
+          reporter.warning(
+            NoPosition(), s"Could not instrument [${tree.getClass.getSimpleName}/${tree.symbol}]."
```

Comment:
This seems to cause a compilation issue;

```
scala.reflect.internal.util.NoPosition.type does not take parameters
```

Reply:
Also any idea what this warning actually looks like when included `NoPosition`?

Reply:
Ooohhh ... my bad. Forgot to commit/push the latest and greatest.